tempesta_fw/cfg.h

```diff
@@ -125,6 +125,7 @@
  */
 typedef struct {
 	const char *fst;
+	const char *fst_ext;
```

Comment:
Please write a comment for the field above.

Reply:
Done.